machine-learning/ml/preprocessing/preprocessing.py

```diff
@@ -73,6 +73,11 @@ def retrieve_labelled_instances(dataset, refactoring: LowLevelRefactoring):
     if USE_PROCESS_AND_AUTHORSHIP_METRICS and not refactoring.refactoring_level() == 'class':
         x = x.drop(["authorOwnership", "bugFixCount", "linesAdded", "linesDeleted", "qtyMajorAuthors",
                     "qtyMinorAuthors", "qtyOfAuthors", "qtyOfCommits", "refactoringsInvolved"], axis=1)
+    else:
```

Comment:
I did not know this `query()` method, interesting!

This `|` means `or`? Shouldn't it be an `&` / `and`?

Reply:
This is inconsistent with the filter you set up earlier, see above.

Reply:
@mauricioaniche in pandas queries you can use the bitwise operators "&", "|" like "and" and "or".
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.query.html

Reply:
I changed it regardless, in order to keep it consistent.